module-cache.ts

```diff
@@ -0,0 +1,6 @@
+import {Browser} from './browser';
+export class ModuleCache {
+    constructor(browser: Browser) { var dictionary = {}; }
```

Comment:
And there should be a blank line between the property cache and the constructor.